prow/jenkins/jenkins_test.go

```diff
@@ -264,16 +288,24 @@ func TestBuildCreate(t *testing.T) {
 				baseURL: ts.URL,
 			}
 
-			jc.BuildFromSpec(testCase.input, "buildID", "prowJobID")
+			buildErr := jc.BuildFromSpec(testCase.input, "buildID", "prowJobID")
+
+			if buildErr != nil {
+				t.Errorf("%s: unexpected build error: %v", testCase.name, buildErr)
+			}
+
+			if !reflect.DeepEqual(testCase.output, actualPaths) {
+				t.Errorf("%s: expected path %s, got %s", testCase.name, testCase.output, actualPaths)
+			}
 		})
 	}
 }
 
-func TestGetJobName(t *testing.T) {
+func TestJobStringBuilding(t *testing.T) {
```

Comment:
I think usually we'd have a test per function

Reply:
No prob, just wanted to minimize the amount of copypaste here.